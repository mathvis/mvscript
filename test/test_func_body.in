func a(b: int, c: string) int {
	let a: int = 0;
}
