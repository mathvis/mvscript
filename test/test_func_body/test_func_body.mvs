func a(b: int, c: string) int {
	let a: int = 0;
	return a;
}
