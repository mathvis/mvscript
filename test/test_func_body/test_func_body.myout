Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",IntT),(VarIdentifier "c",StringT)] IntT (Just (Block (FunctionBlock IntT) [Decl (Variable (VarIdentifier "a") (Just IntT) (Just (Type (Int 0))))])))
