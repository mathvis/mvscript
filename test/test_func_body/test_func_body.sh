#!/bin/bash
helix test_func_body.mvs test_func_body.out --vsplit
