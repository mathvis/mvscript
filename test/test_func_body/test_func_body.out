Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",IntT),(VarIdentifier "c",StringT)] IntT (Just (Block (FunctionBlock IntT) [Decl (Variable (VarIdentifier "a") (Just IntT) (Just (Type (Int 0)))),Expr (Return (Just (VarIdentifier "a")))])))
